.github/workflows/mpet-docker-test.yml

```diff
@@ -33,6 +29,12 @@ jobs:
         fetch-depth: 1
         path: base-mpet
         ref: ${{ github.base_ref }}
+
+    - name: Install additional dependencies using mpet's setup.py
+      run: |
+        cd mpet
+        pip install .[test]
```

Comment:
@v1kko let me know what you think about this solution for installing dependencies.

Reply:
I like it!